hack/shell

```diff
@@ -21,4 +21,4 @@ if [ -n "$MOUNT_BUILDKIT_SOURCE" ]; then
 fi
 
 set -x
-docker run $SSH $volumes -it --privileged -v /tmp --net=host -e BUILDKIT_REGISTRY_MIRROR_DIR=/root/.cache/registry --rm $(cat $iidfile)
+docker run $SSH $volumes -it --privileged -v /tmp --net=host --entrypoint ash -e BUILDKIT_REGISTRY_MIRROR_DIR=/root/.cache/registry --rm $(cat $iidfile)
```

Comment:
```suggestion
docker run $SSH $volumes -it --privileged -v /tmp --net=host --entrypoint sh -e BUILDKIT_REGISTRY_MIRROR_DIR=/root/.cache/registry --rm $(cat $iidfile)
```

IMO the shell type can be generic here.

Reply:
That image is always alpine, so I think it's better to be precise. If it was not alpine, then you wouldn't want to use `sh` but `bash`.